IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -199,6 +199,18 @@ - (MSIDToken *)getATForAccount:(MSIDAccount *)account
     return matchedTokens[0];
 }
 
+- (MSIDAdfsToken *)getADFSTokenWithRequestParams:(MSIDRequestParameters *)parameters
+                                         context:(id<MSIDRequestContext>)context
+                                           error:(NSError **)error
+{
+    if (error)
+    {
+        *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorInvalidInternalParameter, @"ADFS tokens are not currently supported for MSAL", nil, nil, nil, context.correlationId, nil);
```

Comment:
"for MSAL" needs to be changed

Reply:
If you have a better name, let's change it :)